automation/test.sh

```diff
@@ -110,7 +110,7 @@ make cluster-sync
 
 kubectl version
 
-ginko_params="--test-args=--ginkgo.timeout=24h --ginkgo.noColor --junit-output=${ARTIFACTS_PATH}/junit.functest.xml"
+ginko_params="--test-args=--ginkgo.timeout=24h --ginkgo.noColor --ginkgo.junit-report=${ARTIFACTS_PATH}/junit.functest.xml"
```

Comment:
note that ARTIFACT_DIR might need to be supplied in other argument,
at least this what i was the state at the past
otherwise it didnt work well

Reply:
I don't understand what you're saying

Reply:
there was a problem in the past that `ARTIFACTS_PATH` had to be supplied with --output-dir or so
else it won't work if it is part of the junit-report
but if it works for you, then all good already